content/en/docs/setup/independent/high-availability.md

```diff
@@ -232,218 +173,158 @@ done
-              advertise-client-urls: "https://CP1_IP:2379"
-              listen-peer-urls: "https://CP1_IP:2380"
-              initial-advertise-peer-urls: "https://CP1_IP:2380"
-              initial-cluster: "CP0_HOSTNAME=https://CP0_IP:2380,CP1_HOSTNAME=https://CP1_IP:2380"
-              initial-cluster-state: existing
-            serverCertSANs:
-              - CP1_HOSTNAME
-              - CP1_IP
-            peerCertSANs:
-              - CP1_HOSTNAME
-              - CP1_IP
-        networking:
+          networking:
             # This CIDR is a calico default. Substitute or remove for your CNI provider.
             podSubnet: "192.168.0.0/16"
 
-1.  Replace the following variables in the template with the appropriate values for your cluster:
+    - `kubernetesVersion` should be set to the Kubernetes version to use. This
+  example uses `stable`.
+    - `controlPlaneEndpoint` should match the address or DNS and port of the load balancer.
+    - It's recommended that the versions of kubeadm, kubelet, kubectl and Kubernetes match.
 
-    - `LOAD_BALANCER_DNS`
-    - `LOAD_BALANCER_PORT`
-    - `CP0_HOSTNAME`
-    - `CP0_IP`
-    - `CP1_HOSTNAME`
-    - `CP1_IP`
-
-1.  Move the copied files to the correct locations:
+1.  Make sure that the node is in a clean state:
 
     ```sh
-    USER=ubuntu # customizable
-    mkdir -p /etc/kubernetes/pki/etcd
-    mv /home/${USER}/ca.crt /etc/kubernetes/pki/
-    mv /home/${USER}/ca.key /etc/kubernetes/pki/
-    mv /home/${USER}/sa.pub /etc/kubernetes/pki/
-    mv /home/${USER}/sa.key /etc/kubernetes/pki/
-    mv /home/${USER}/front-proxy-ca.crt /etc/kubernetes/pki/
-    mv /home/${USER}/front-proxy-ca.key /etc/kubernetes/pki/
-    mv /home/${USER}/etcd-ca.crt /etc/kubernetes/pki/etcd/ca.crt
-    mv /home/${USER}/etcd-ca.key /etc/kubernetes/pki/etcd/ca.key
-    mv /home/${USER}/admin.conf /etc/kubernetes/admin.conf
+    sudo kubeadm reset -f
+    sudo kubeadm init --config=kubeadm-config.yaml
+    ```
+    
+    You should see something like:
+    
+    ```sh
+    ...
+    You can now join any number of machines by running the following on each node
+    as root:
+    
+    kubeadm join 192.168.0.200:6443 --token j04n3m.octy8zely83cy2ts --discovery-token-ca-cert-hash    sha256:84938d2a22203a8e56a787ec0c6ddad7bc7dbd52ebabc62fd5f4dbea72b14d1f
     ```
 
-1.  Run the kubeadm phase commands to bootstrap the kubelet:
+1.  Copy this output to a text file. You will need it later to join other control plane nodes to the cluster.
+
+1.  Type the following and watch the pods of the components get started:
 
     ```sh
-    kubeadm alpha phase certs all --config kubeadm-config.yaml
-    kubeadm alpha phase kubelet config write-to-disk --config kubeadm-config.yaml
-    kubeadm alpha phase kubelet write-env-file --config kubeadm-config.yaml
-    kubeadm alpha phase kubeconfig kubelet --config kubeadm-config.yaml
-    systemctl start kubelet
+    watch kubectl get pods --all-namespaces
```

Comment:
Any reason to not use `kubectl get pod -n kube-system -w` here instead?

Reply:
ok, will switch to -w.
they are pretty much the same.